component/Alerts/styles.css

```diff
@@ -0,0 +1,96 @@
+
+@import url('/styles/nav.css');
+@import url('/styles/styles-global/root.css');
+@import url('/styles.css');
+
+:root{
+    --alert-success-color: #b7e1b4;
+    --alert-info-primary-color: #9999ff;
+    --alert-info-secondary-color: #ffdec5;
+    --alert-error-color: #f8c1c1;
+    --alert-warning-color: #f8f6c4;
+}
+
+.alert-body{
+    font-family: var(--font-family-content);
+    grid-area: main;
+    margin: 0rem 18rem;
+    background-color: var(--white-color);
+    padding: 2rem 2rem;
+}
+.heading{
+    margin: 1rem 0rem;
+    font-size: var(--heading-size);
+    text-align: left;
+    font-weight: var(--heading-weight);
+}
+
+.alert-subheading{
+    margin: 2rem 1rem;
+    font-size: var(--sub-heading-size);
+    text-align: left;
+    font-weight: var(--sub-heading-weight);
+}
+
+
+.alert-content{
+    padding-top: 1rem;
+    font-size: var(--content-size);
+    font-weight: var(--content-weight);
+}
+
+.alt-ty-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    gap: 10px;
+    align-items: center;
+    background-color: var(--primary-vlight-color);
+    padding: 1rem 0rem;
+}   
+
+.alt-child{
+    width: 60%;
+    padding: 1rem 0rem;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-weight: 500;
+}
+
+.alert-suc{
+    background-color: var(--alert-success-color);
+    
+}
+
+.alert-p-info{
+    background-color: var(--alert-info-primary-color);
+}
+
+.alert-s-info{
+    background-color: var(--alert-info-secondary-color);
+}
+
+
+.alert-warn{
+    background-color: var(--alert-warning-color);
+}
+
+.alert-err{
+    background-color: var(--alert-error-color);
+}
+
+.iframe-container{
+    padding-top: 3rem;
+}
+
+iframe{
```

Comment:
If you are using iframes I believe by default it is inline styled so keep it inline styled here itself because it won't be responsive when you shrink the screen size and also give its width as 100%.